lib/ios-fs-helpers.js

```diff
@@ -318,6 +319,7 @@ export async function pushFolder(afcService, srcRootPath, dstRootPath, opts = {}
           throw new TimeoutError(`Timed out after ${elapsedMs} ms`);
         }
       }
+      // @ts-ignore
```

Comment:
why this change is needed?

Reply:
>> fix the original typing issue if possible 

Since the issue occurred with no code change with reference to the current work, we ignored it. Will fix it in separate PR.

Reply:
for sure